README.md

```diff
@@ -219,6 +215,18 @@ $ ls bin/local
 ecs-cli
 ```
 
+### Adding a new dependency
```

Comment:
Good idea! I think linking to the main github page should probably be enough -- considering we're hopefully going to move away from it to use `dep` instead, might not be worth investing too much time in documenting how to use godep.